microprofile/microprofile-lra/coordinator/src/main/java/org/wildfly/extension/microprofile/lra/coordinator/MicroProfileLRACoordinatorAdd.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * JBoss, Home of Professional Open Source.
+ * Copyright 2023, Red Hat, Inc., and individual contributors
+ * as indicated by the @author tags. See the copyright.txt file in the
+ * distribution for a full listing of individual contributors.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+
+package org.wildfly.extension.microprofile.lra.coordinator;
+
+import org.jboss.as.controller.AbstractBoottimeAddStepHandler;
+import org.jboss.as.controller.CapabilityServiceBuilder;
+import org.jboss.as.controller.OperationContext;
+import org.jboss.as.controller.OperationFailedException;
+import org.jboss.dmr.ModelNode;
+import org.jboss.msc.service.ServiceController;
+import org.jboss.msc.service.ServiceName;
+import org.jboss.tm.XAResourceRecoveryRegistry;
+import org.wildfly.clustering.service.CompositeDependency;
+import org.wildfly.clustering.service.ServiceSupplierDependency;
+import org.wildfly.clustering.service.SupplierDependency;
+import org.wildfly.extension.microprofile.lra.coordinator._private.MicroProfileLRACoordinatorLogger;
+import org.wildfly.extension.microprofile.lra.coordinator.service.LRACoordinatorService;
+import org.wildfly.extension.microprofile.lra.coordinator.service.LRARecoveryService;
+import org.wildfly.extension.undertow.Capabilities;
+import org.wildfly.extension.undertow.Host;
+import org.wildfly.extension.undertow.UndertowService;
+
+class MicroProfileLRACoordinatorAdd extends AbstractBoottimeAddStepHandler {
+    static MicroProfileLRACoordinatorAdd INSTANCE = new MicroProfileLRACoordinatorAdd();
+
+    private MicroProfileLRACoordinatorAdd() {
+    }
+
+    @Override
+    protected void populateModel(ModelNode operation, ModelNode model) throws OperationFailedException {
+        MicroProfileLRACoordinatorLogger.LOGGER.trace("MicroProfileLRACoordinatorAdd.populateModel");
+        MicroProfileLRACoordinatorSubsystemDefinition.HOST.validateAndSet(operation, model);
+        MicroProfileLRACoordinatorSubsystemDefinition.SERVER.validateAndSet(operation, model);
+    }
```

Comment:
Please pass these 2 AttributeDefinitions to the superclass constructor and remove this method. Other than the trace logging this method does what the superclass does if it know what the attributes are.